misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -31,7 +31,7 @@ class AwsSqsJobQueueModule(
     bind<TransactionalJobQueue>().to<SqsTransactionalJobQueue>()
     multibind<Service>().to<SqsJobConsumer>()
 
-    install(ExecutorServiceModule.withCachedThreadPool(ForSqsConsumer::class, "sqs-consumer-%d"))
+    install(ExecutorServiceModule.withFixedThreadPool(ForSqsConsumer::class, "sqs-consumer-%d", 4))
```

Comment:
This seems like it should be configurable

Reply:
Maybe as part of `AwsSqsJobQueueConfig`?

Reply:
If it was unbounded before I think making it configurable is a separate PR.